packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -44,6 +44,8 @@ import {
 import { MCULayoutEventHandler } from './LayoutHandler';
 import Call from './Call';
 
+const SDK_VERSION = '2.5.1';
```

Comment:
Shouldn't this variable be dynamic @DeividVeloso? maybe getting from the package.json 🤔

Reply:
That was to add a quick way to get the version, but as it was not deployed yet I will add dynamic it will need to change some more code.

Reply:
yes, it would be better if is automatic